packages/api/emails/survey/html.pug

```diff
@@ -0,0 +1,106 @@
+            }
+            .f1 h6, .f1 p {
+                margin-bottom: 0px;
+                margin-top: 0px;
+                color: red;
+            }
+            .f1 {
+                margin-top: 50px;
+            }
+
+    body
+        img(src= `${appUrl}/email/header.jpg` alt='header' style='width:100%; height:auto;')
+        //- for redering header image in test, use code bellow, for prodution use above.
+        //- img(src= "https://ksf-stg.herokuapp.com/email/header.jpg" alt='header' style='width:100%; height:auto;')
+        div(class="container")
+            h1 FINAL STEP: SUPPORTING SURVEY
+            
+            p Hi #{name},
+            
+            p If you are receiving this message, both your online application and HIPAA Authorization Form have been processed in our system and we are interested in learning more about your family’s current medical circumstance.
+            p 
+                | This is the final step of the nomination process in which the patient’s family is involved. All future communications will occur between Keep Swimming Foundation and the member of your medical staff who nominated your family. The final decision regarding the families who will be provided grants will be 
+                strong announced in April 2020.
+            p 
+                | Below you will find a link to Keep Swimming Foundation’s Supporting Survey. In order to be considered for a Keep Swimming Foundation grant, we need either the patient or a member of their Immediate Family to complete our Supporting Survey and submit it to Keep Swimming Foundation by the deadline, which is provided below. 
+                strong In addition, your nominating healthcare provider (social worker, nurse, physician) must sign off on your completed submission.
+            p 
+                strong Please Note: 
+                | If you do not have the appropriate technology (scanner, computer, printer, etc. to complete this submission, your hospital's social worker may be able to assist you.
+            p While the survey is designed to be very simple, if you have any questions about our Supporting Survey, please do not hesitate to send us your questions via email at: info@keepswimmingfoundation.org. 
+            p The below link will take you to a Dropbox file. Simply click the "Download" box on the top right and then select "Direct Download" to complete the document on your computer.
+            p We look forward to finalizing this nomination process with you. Thank you for your time.
+            
+            p Sincerely,
+
+            p -Keep Swimming Foundation
+
+
+            h3 DEADLINE:
+
+            p 
+                strong Deadline: Tuesday, March 31, 2020 at 11:59 PM CT 
+                | (All Supporting Surveys submitted after this time will not be considered). The  completion of this Supporting Survey does not guarantee the nominated individual family a Keep Swimming Foundation grant. 
```

Comment:
Looks like there's an extra space between `the` and `completion`.

Reply:
Thank you done